packages/renderer/src/lib/chat/components/chat.svelte

```diff
@@ -106,17 +117,24 @@ const hasModels = $derived(models && models.length > 0);
   {/if}
   <div class="flex min-h-0 flex-1">
         {#if hasModels}
-            <div class="flex flex-col flex-3/4"> 
+            <div class="flex flex-col flex-3/4">
                 <Messages
                     {readonly}
                     loading={chatClient.status === 'streaming' || chatClient.status === 'submitted'}
                     messages={chatClient.messages}
                 />
-                <form class="bg-background mx-auto flex w-full gap-2 px-4 pb-4 md:max-w-3xl md:pb-6">
+                <form class="bg-background mx-auto flex w-full gap-2 px-4 md:max-w-3xl">
                     {#if !readonly}
                         <MultimodalInput {attachments} {chatClient} {selectedModel} {selectedMCP} bind:mcpSelectorOpen={mcpSelectorOpen} class="flex-1" />
                     {/if}
                 </form>
+                {#if !readonly}
```

Comment:
suggestion(non-blocking): introduce a small Svelte component for the token count